.github/workflows/release.yml

```diff
@@ -4,46 +4,197 @@ on:
     tags:
       # this is a glob, not a regexp
       - '[0-9]*'
+
+
 jobs:
-  cockpituous:
+  source:
     runs-on: ubuntu-latest
-    environment: release
     container:
-      image: ghcr.io/cockpit-project/release
+      image: ghcr.io/cockpit-project/unit-tests
+      options: --user root
```

Comment:
Why do we need root here? This is normally done to make it easier to share files between the host and the container, but that's not what happens here.

Reply:
Same story here as cockpit-projects/bots#3365.  It's standard practice.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

says

> `/github/workspace` - **Note**: GitHub Actions must be run by the default Docker user (root). Ensure your Dockerfile does not set the `USER` instruction, otherwise you will not be able to access `GITHUB_WORKSPACE`.